app/views/batch/_metadata.html.erb

```diff
@@ -28,14 +28,14 @@
       <div class="col-sm-6">
       <h3>Applies to all files just uploaded</h3>
       <%= f.input :resource_type, as: :select_with_help, collection: Sufia.config.resource_types,
-          input_html: { class: 'form-control', multiple: true, required: true } %>
+          input_html: { class: 'form-control', multiple: true } %>
 
-      <%= f.input :tag, as: :multi_value_with_help, input_html: { required: true } %>
+      <%= f.input :tag, as: :multi_value_with_help %>
 
-      <%= f.input :creator, as: :multi_value_with_help, input_html: { required: true } %>
```

Comment:
But that incantation does not affect whether it shows on the form as required or not. The input sets the required attribute if the field is required on the form. See: https://github.com/projecthydra/sufia/blob/master/app/forms/sufia/forms/generic_file_edit_form.rb#L6